packages/sdks/spl/src/getStakePool.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * TODO: FIX DEPENDENCY CONFLICTS TO PREVENT HAVING TO DO THIS.
```

Comment:
If we upgrade spl-token across the board will it solve this? I think marinade-sdk uses an old version but was recently updated to use @solana/spl-token-3.x

Reply:
I tried fixing it but ran into multiple issues and so just left it like this to save time. It's probably fixable though